part-iv-advanced-tuning/performance-tuning.md

```diff
@@ -122,7 +122,22 @@ See [Using Snapshot Events](../part-iii-infrastructure-components/repository-and
 
 ### Filtering Snapshots
 
-Sometimes a snapshot becomes obsolete (the Aggregate structure has changed since it was snapshotted). In those cases it is convenient to filter snapshots. This is where `SnapshotJury` comes in place. It decides based on `DomainEventData` whether a snapshot should be taken into processing. If none provided, implementation (`NoSnapshotJury`) which returns always `true` is used. `RevisionBasedSnapshotJury` can be used as an alternative provided by Axon which checks the snapshot based on Aggregate revision. `SnapshotJury` has to be provided to the `EventStore`. If required, a custom implementation can be provided.  
+Sometimes a snapshot becomes obsolete (the Aggregate structure has changed since it was snapshotted). In those cases it is convenient to filter snapshots. This is where `snapshotFilter` comes in place. It is a Java `Predicate` which decides based on `DomainEventData` whether a snapshot should be taken into processing. If none provided, implementation which returns always `true` is used. `snapshotFilter` has to be provided to the `EventStorageEngine`:
```

Comment:
Suggestions/Typos:
- `...should be taken into processing.` -> `...should be taken into *account for* processing.`
- `If none provided, implementation...` -> `If none provided, *an* implementation...`